TrackingTools/Configuration/python/TrackingTools_EventContent_cff.py

```diff
@@ -9,7 +9,7 @@
 
 from Configuration.Eras.Modifier_phase2_hgcal_cff import phase2_hgcal
 phase2_hgcal.toModify( TrackingToolsAOD,
-                       outputCommands = TrackingToolsAOD.outputCommands + ['keep recoGsfTracks_electronGsfTracksFromMultiCl_*_*'])
+                       outputCommands = TrackingToolsAOD.outputCommands + ['keep recoGsfTracks_electronGsfTracks_*_*'])
```

Comment:
this part is not needed anymore (`recoGsfTracks_electronGsfTracks_` are a part of the standard content)

Reply:
Done.